### ISSUE TITLE
Linking to tacopie (libtacopie.a)

### ISSUE THREAD

**Issue by julianoes** [Cylix/cpp_redis#76]
Hi all,

First of all: thanks for the good docs and clean project, very refreshing!

I installed cpp_redis as described in https://github.com/Cylix/cpp_redis/wiki/Mac-&-Linux-Install
Now I ran into a two linking problems regarding tacopie.

1. I forgot to link to it at first even though it's clearly written in the wiki, however, it is not included in the [example CMakelists.txt](https://github.com/Cylix/cpp_redis/blob/master/examples/CMakeLists.txt#L47).

2. When I realized that I needed to link to tacopie, I found that it is not installed into `/usr/local/lib`. My current workaround is:
    ```
    target_link_libraries(mywhatever
        cpp_redis
        $ENV{HOME}/src/cpp_redis/deps/lib/libtacopie.a
        pthread
    )
    ```

What am I doing wrong?

**Comment by Cylix** [comment]
Hi,

Thanks for opening this issue and reporting the issues you faced!

Concerning the first point, the issue is mostly that CMake does some deduction since the examples directory is a subdirectory of the project. I Decided to leave it like this as actually this CMakefile is only intended to provide the compilation of examples, but not to be an example of the compilation steps itself.

For the second point, it was a bit tricky to fix because of the way add_externalproject directive of CMake works... So I switched it back to add subdirectory and it seems to work fine :)

I deployed the change and I close this issue, but do not hesitate to let me know if something is still wrong!

Thanks again :)

Best
